services/cannon/src/Cannon/WS.hs

```diff
@@ -234,6 +237,23 @@ sendMsgIO :: L.ByteString -> Websocket -> IO ()
 sendMsgIO m c = do
   recoverAll retry3x $ const $ sendBinaryData (connection c) m
 
+sendMsgConduit :: Key -> Websocket -> ConduitT ByteString Void (ResourceT WS) ()
```

Comment:
Looks like this replaces `sendMsg`, so that function can be deleted.

Reply:
Indeed, `sendMsg` is a left-over.